pyaedt/application/Analysis3D.py

```diff
@@ -1034,3 +1034,86 @@ def flatten_3d_components(self, component_name=None, purge_history=True, passwor
         if not self.design_type == "Icepak":
             self.mesh._refresh_mesh_operations()
         return True
+
+    def identify_touching_conductors(self, object_name=None):
+        # type: (str) -> dict
+        """Identify all touching components and group in a dictionary. This method requires that
+        the ``pyvista`` package is installed.
+
+        Parameters
+        ----------
+        object_name : str, optional
+            Starting object to check for touching elements. The default is ``None``.
```

Comment:
```suggestion
            Starting object to check for touching elements. The default is ``None``.
            
```
Blank line is necessary between sections for proper doc rendering.